lib/Support/Pipeline.cpp

```diff
@@ -127,6 +127,7 @@ void MappingTraits<offloadtest::Buffer>::mapping(IO &I,
     DATA_CASE(Int16, int16_t)
     DATA_CASE(Int32, int32_t)
     DATA_CASE(Int64, int64_t)
+    DATA_CASE(Float16, uint16_t)
```

Comment:
Do we want this to be unit16_t or hex16?

Seems like hex16 is probably more useful.

Reply:
The reference code from DXC uses uint16_t to represent half, if that matters.
https://github.com/microsoft/DirectXShaderCompiler/blob/624665f3987d379a299aa14dbc53e0cd3b96afea/include/dxc/Test/HlslTestUtils.h#L567